### PR TITLE
added solutions to big_o_exercises

### PR DESCRIPTION
**Pull request by aykaypkt**
My solutions to big_o_exercises

### CODE REVIEW

File: big_o_exercise/readme.md [hunk 1]
```diff
@@ -4,16 +4,16 @@
...
-5. `O(n + n + n + n)`
-6. `O(1000 * log(n) + n)`
-7. `O(1000 * n * log(n) + n)`
-8. `O(2^n + n^2)`
-9. `O(5 + 3 + 1)`
-10. `O(n + n^(1/2) + n^2 + n * log(n)^10)`
+1. `O(n + 10)` // O(n)
+2. `O(100 * n)` // O(n)
+3. `O(25)` // O(1)
+4. `O(n^2 + n^3)` // O(n^2 )
```
Comment (mmmaaatttttt) [inline big_o_exercise/readme.md]: Careful here, n^2 is dominated by n^3

File: big_o_exercise/readme.md [hunk 2]
```diff
@@ -4,16 +4,16 @@
...
+5. `O(n + n + n + n)` // O(n)
+6. `O(1000 * log(n) + n)` // O(log(n))
```
Comment (mmmaaatttttt) [inline big_o_exercise/readme.md]: Which grows faster, log(n) or n?

File: big_o_exercise/readme.md [hunk 3]
```diff
@@ -4,16 +4,16 @@
...
+7. `O(1000 * n * log(n) + n)` // O(n)
```
Comment (mmmaaatttttt) [inline big_o_exercise/readme.md]: Which grows faster, n or n log(n)?

Comment (aykaypkt) [inline big_o_exercise/readme.md]: Thanks, 
For some reason I kept thinking 
O(n) > O(nlog(n))> O(log(n)) which is why my answers were consistently wrong

**Comment by mmmaaatttttt** [comment]
Looks great!

**Pull request closed without merging**